FordLabs/retroquest
# Add login recovery settings buttons & confirmation modals

## What was done
- [x] Added links in the Settings>Account modal to allow users to get sent an email to reset their password or reset their email addresses
- [x] Added confirmation modals for either of those buttons so the user knows their email was sent succesfully 

### Demo
<img width="582" alt="Screen Shot 2022-10-13 at 1 18 39 PM" src="https://user-images.githubusercontent.com/17144844/195665197-2eead5a1-7c0b-47ff-93cc-f610c33fc7f0.png">
<img width="573" alt="Screen Shot 2022-10-13 at 1 37 54 PM" src="https://user-images.githubusercontent.com/17144844/195667150-fe5c8f39-1704-41a4-963a-a94ba0e2af79.png">
<img width="568" alt="Screen Shot 2022-10-13 at 1 38 02 PM" src="https://user-images.githubusercontent.com/17144844/195667156-bf68cd06-80a0-4d8d-8261-c71d4039a862.png">

## Testing Instructions
1) Go to Settings>Account for a team that has at least one email
2) Click on the "Send Password Reset Link" link and ensure it makes an api call or two (one for each email the team has) and shows the corresponding confirmation modal. Clicking close on that modal should close the entire modal.
3) Click on the "Send Board Owner Update Link" link and ensure it makes an api call or two (one for each email the team has) and shows the corresponding confirmation modal. Clicking close on that modal should close the entire modal.
